### PR TITLE
FCBH-1900 Backend Download Storage Filling up

### PR DESCRIPTION
**Pull request by mgonzalezs**
# Description
- Created a downloads folder for the temporary files
- Added `php artisan delete:temporaryZipFiles` command to delete files with more than 1 hour of creation
- Added DeleteTemporaryZipFiles command to the cron jobs, every hour will be executed

## Issue Link
[FCBH-1900](https://fullstacklabs.atlassian.net/browse/FCBH-1900) 

## How Do I QA This
- Run `php artisan delete:temporaryZipFiles` command and verify that all the files older than 1 hour on the `public/downloads` folder are deleted

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed locally.

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -72,6 +74,11 @@ protected function schedule(Schedule $schedule)
             ->hourly()
             ->onOneServer()
             ->withoutOverlapping();
+
+        $schedule->command(DeleteTemporaryZipFiles::class)
+            ->hourly()
+            ->onOneServer()
```
Comment (jrstear) [inline app/Console/Kernel.php:80]: @mgonzalezs this needs to run on each server, so i believe it is incorrect to use onOneServer?

File: app/Console/Commands/DeleteTemporaryZipFiles.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    public function handle()
+    {
+        $this->alert(Carbon::now() . ': temporary zip files deletion started.');
+        $public_dir = public_path() . '/downloads';
+
+
+        if (File::exists($public_dir)) {
+            $files = collect(File::files($public_dir));
+
+            $paths = $files->filter(function ($file) {
```
Comment (jrstear) [inline app/Console/Commands/DeleteTemporaryZipFiles.php:39]: i would prefer that only files ending in .zip be removed.  leaving it wide open may have unexpected effects later.

File: app/Console/Kernel.php (hunk 1 above)
Comment (mgonzalezs) [reply app/Console/Kernel.php:80]: You are right. Fixed

File: app/Console/Commands/DeleteTemporaryZipFiles.php (hunk 2 above)
Comment (mgonzalezs) [reply app/Console/Commands/DeleteTemporaryZipFiles.php:39]: Fixed Jon

**Pull request merged**